<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignupPage</class>
 <widget class="QWidget" name="SignupPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="signupAccount">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Your Account</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="signupPassword">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Your Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="signupPasswordConfirm">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Confirm your password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pokemonName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Your Pokemon Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generatePokemon">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>生成</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>98</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorinfo">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>361</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(204, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="signupNickName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Your NickName</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
